src/osd/scrubber/PrimaryLogScrub.cc

```diff
@@ -87,9 +87,8 @@ void PrimaryLogScrub::submit_digest_fixes(const digests_fixes_t& fixes)
 
 
     ctx->register_on_success([this]() {
-      dout(20) << "updating scrub digest " << num_digest_updates_pending
-               << dendl;
-      if (--num_digest_updates_pending <= 0) {
+      if ((num_digest_updates_pending >= 1) &&
```

Comment:
Is it possible for this portion of the condition to not be met?

Reply:
Yes. It does happen. I am not sure I understand all the scenarios that lead to that, but the ones I know were a result
of bugs already fixed in newer Ceph versions (not all backported - I will be investing time in that).

Reply:
This seems to have been removed, presumably no longer needed?

Reply:
Was already incorporated separately, to facilitate backporting.

Reply:
Got it.